tools/functional/api/test_import_export.py

```diff
@@ -0,0 +1,61 @@
+import time
+
+
+def test_group_import_export(gl, group, temp_dir):
+    export = group.exports.create()
+    assert export.message == "202 Accepted"
+
+    # We cannot check for export_status with group export API
+    time.sleep(10)
+
+    import_archive = temp_dir / "gitlab-group-export.tgz"
+    import_path = "imported_group"
+    import_name = "Imported Group"
+
+    with open(import_archive, "wb") as f:
+        export.download(streamed=True, action=f.write)
+
+    with open(import_archive, "rb") as f:
+        output = gl.groups.import_group(f, import_path, import_name)
+    assert output["message"] == "202 Accepted"
+
+    # We cannot check for returned ID with group import API
+    time.sleep(10)
+    group_import = gl.groups.get(import_path)
+
+    assert group_import.path == import_path
+    assert group_import.name == import_name
+
+
+def test_project_import_export(gl, project, temp_dir):
+    export = project.exports.create()
+    export.refresh()
```

Comment:
It was probably there originally because the create/POST request only gets a `"message": "202 Accepted"` response so there's no initial `export_status` for the loop below, I will check. Maybe a for loop with a break and only one `refresh` would be more elegant though :)

Reply:
It was needed but I placed an explicit get here instead, maybe nicer :)
```diff
 def test_project_import_export(gl, project, temp_dir):
     export = project.exports.create()
-    export.refresh()
+    assert export.message == "202 Accepted"
+
+    export = project.exports.get()
+    assert isinstance(export, gitlab.v4.objects.ProjectExport)

     count = 0
     while export.export_status != "finished":
```